src/app/countries.ts

```diff
@@ -1030,3 +1030,6 @@ export const countryOptions: { label: string; value: string }[] = COUNTRIES.map(
 }));
 
 export type countryISO2 = (typeof COUNTRIES)[number]['iso2'];
+
+export const codeeToCountryName = (iso2Code: string) =>
```

Comment:
```suggestion
export const codeToCountryName = (iso2Code: string) =>
```
and related imports etc.